components/authn-service/tokens/pg/pg.go

```diff
@@ -8,11 +8,13 @@ import (
 	"go.uber.org/zap"
 
 	// adapter for database/sql
-	_ "github.com/lib/pq"
-
+	authz_v2 "github.com/chef/automate/api/interservice/authz/v2"
 	tokens "github.com/chef/automate/components/authn-service/tokens/types"
 	platform_config "github.com/chef/automate/lib/platform/config"
 	"github.com/chef/automate/lib/tls/certs"
+
+	// TODO do we need this?
```

Comment:
Does it work without? 😉 
See also [its docs](https://godoc.org/github.com/lib/pq); the reason for importing this with an underscore is that
- we need the postgres code to be loaded
- we don't use the package in any way, we only use `database/sql`

Hence the underscore import. 😃 

For the curious, the registration stuff happens here: https://github.com/lib/pq/blob/3427c32cb71afc948325f299f040e53c1dd78979/conn.go#L51-L53 -- so when the package is imported like this, its `init` function will be run and the `database/sql` package suddenly knows now to speak PG.